src/cpp.cc

```diff
@@ -1505,8 +1602,18 @@ void Printer::Cpp::print_table_init(const AST &ast) {
     if (ast.window_mode) {
       stream << " opts.window_size, opts.window_increment, ";
     }
-    stream << "\""<< i->second->table_decl->name() << "\");" << endl;
+
+    stream << "\""<< i->second->table_decl->name() << "\"";
+    if (ast.checkpoint) {
+      stream << ", opts.checkpoint_out_path," << endl;
+      stream << indent() << "                opts.checkpoint_in_path, "
```

Comment:
True. But since I also need to debug the generated code and I don't want to have to scroll to read the whole line, I sometimes print the generated code on multiple lines.